### PR TITLE
[wger-project/flutter#306] Create text prompt for empty screens

### PR DESCRIPTION
**Pull request by freddyc2003**
# Proposed Changes

- Text displayed to the user when screens are empty prompting them to use a particular feature of the app,
- wording is generic across the Workout plans, Nutritional plans and Gallery screens to facilitate easy translation.

### CODE REVIEW

File: lib/widgets/core/text_prompt.dart [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class TextPrompt extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(
+            'Ready to start?',
```
Comment (rolandgeider) [inline lib/widgets/core/text_prompt.dart:11]: Can you mark these strings as translatable? Just add them to app_en.arb and then you can access the key with `AppLocalizations.of(context).yourKey`

Comment (rolandgeider) [reply lib/widgets/core/text_prompt.dart:11]: (the generated files are created on every build but sometimes the IDE still can't recognize them and mark the AppLocalizations key, but if it builds, just ignore the error)

Comment (freddyc2003) [reply lib/widgets/core/text_prompt.dart]: I have marked the strings as translatable and commited the changes.

Comment (rolandgeider) [reply lib/widgets/core/text_prompt.dart]: thanks, looks good! Will run some tests later today and then we can merge this :)

**Comment by rolandgeider** [comment]
Merged, thanks!

**Pull request merged**